### ISSUE TITLE
Please add documentation to aur.py

### ISSUE THREAD

**Issue by panchoh** [kewlfft/ansible-aur#6]
The documentation only exists on `README.md`.

When I try:
```
$ ansible-doc aur
ERROR! module aur missing documentation (or could not parse documentation): Parsing produced an empty object.
```
I use this a lot from my editor, to check the params of the modules without having to fire up a browser...

Thanks a bunch!

BTW, If you are busy with other stuff, I could try and do it and drop you a PR, but not immediately.

**Comment by kewlfft** [comment]
please have a look at the documentation added and let me know

**Comment by panchoh** [comment]
Perfect.

Very much appreciated.

**Comment by kewlfft** [comment]
If you can test it and let me know if everything is ok, I will release 0.20 in the coming hours

**Comment by panchoh** [comment]
I've battle tested it with my ansible installer, and it works like a charm.
I've fixed a couple of minor things in the doc (#7), but besides those, it's golden.
